tests/test_hookman_utils.py

```diff
@@ -8,3 +8,16 @@ def test_find_config_files(datadir):
 
     config_files = find_config_files([datadir / 'non_existing_folder'])
     assert len(config_files) == 0
+
+
+def test_load_shared_lib(simple_plugin_dll):
+    import os
+    import sys
+    from hookman.hookman_utils import load_shared_lib
+
+    if sys.platform.startswith('win'):
```

Comment:
It's better to use `skipif` to skip the test (e.g., when it's not on 'win' platform)

Reply:
Yep, in this case:

```python
@pytest.mark.skipif(sys.platform.startswith("win"), reason='path only needs changing on Windows')
def test_path_change_when_loading_shared_lib(simple_plugin_dll):
    ...
```

Also note my suggestion about the test name.  😉

Reply:
Thanks for helping the `skipif` usage.